test/PolymerDom_test.html

```diff
@@ -0,0 +1,21 @@
+<!doctype html>
+<html>
+  <head>
```

Comment:
First thing in the head should be the web components polyfill (`<script src="packages/web_components/webcomponents-lite.js></script>`). That should get firefox and safari working, if it doesn't then we may also need to wait until polymer is ready before running the tests.